### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Denpeer**
User Story:
As a developer I want my code to be properly tested so that continious integration is possible.

Note that the perceptTest made travis fail. This did not happen loccaly as expected. We removed this test to the test agent so travis does not run the test.

### CODE REVIEW

**Comment by klikooo** [comment]
I approve but I wrote this code.

File: src/DUWOtestAgent/events.mod2g [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
 
 module events {
 	
+	%Should be removed soon, temporary demo code.
+	if bel(buildableStudent(X,Y),
+		buildableStudentList(Z),
+		not(member(X,Z))) 
+	then delete(buildableStudentList(Z)) 
+		+ insert(buildableStudentList([X|Z])).
+
```
Comment (MarcianoJ) [inline src/DUWOtestAgent/events.mod2g:11]: Looks like this shouldnt be here

Comment (Denpeer) [inline src/DUWOtestAgent/events.mod2g:11]: You're right, should be fixed now

**Comment by MarcianoJ** [comment]
Approved

**Pull request merged**